Changelog.md

```diff
@@ -7,6 +7,17 @@ This project does _not_ adhere to
 [Semantic Versioning](https://semver.org/spec/v2.0.0.html) but contrary to it
 every new version is a new major version.
 
+## 88 - 2023-08-15
```

Comment:
```suggestion
## 88 - 2023-08-17
```

Reply:
Thanks for the suggestion. Fixes as per your orders : )